modules/programs/direnv.nix

```diff
@@ -63,10 +71,14 @@ in {
       '';
     };
 
-    enableNixDirenvIntegration = mkEnableOption ''
-      <link
-          xlink:href="https://github.com/nix-community/nix-direnv">nix-direnv</link>,
-          a fast, persistent use_nix implementation for direnv'';
+    nix-direnv = {
+      enable = mkEnableOption ''
+        <link
+            xlink:href="https://github.com/nix-community/nix-direnv">nix-direnv</link>,
+            a fast, persistent use_nix implementation for direnv'';
+      enableFlakes = mkEnableOption "Flake support in nix-direnv";
```

Comment:
Is it worth making this `flakes.enable` for future extensibility? (I don't know a ton about flakes, so that may not be an issue.)

Reply:
I don't think so to be honest. Presumably at some point flakes will stop being experimental and then nix-direnv won't require you to opt-in to using nixUnstable, and this option can be dropped. IMO the option name is clear and matching up with what nix-direnv calls it is both unsurprising to the user and lets me use `inherit` on L93 to be a little more concise.